docs/use_cases.md

```diff
@@ -3,7 +3,61 @@
 Here are examples of questions this tool can help to answer,
 types of results it can give, and other use cases.
 
-## Synthetic F280 records
+## What is the approximate contamination rate of a set of inspected
+consignments?
```

Comment:
Headings needs to be a single line in the source code. (same for *what are the trade offs between effort and effectiveness for*)